### DISCUSSION TITLE
chart vs slice vs form_data

### DISCUSSION THREAD

**Discussion by JakobMiksch** [apache/superset#20230] [Q&A / Help]
Is there any official definition what `chart`, `slice` and `form_data` is and how they relate to each other?

I am using the API and find the meanings of the terms quite confusing.

**Comment by artemonsh** [comment]
+1

**Comment by renwei-work** [comment]
+1 
chart seems same as slice, not sure if slice was evolved from chart?
form_data seems a snapshot of a chart, but what are the detail information in it and how to generate it from API?

**Comment by Always-prog** [comment]
+1 here

**Comment by Always-prog** [comment]
Maybe the word chart refers to a full-fledged saved chart, whereas slice can only exist in the constructor without being saved or added to any dashboard. 

form_data is just options of chart from controlPanel, as @renwei-work say it
